codetransformer/transformers/patterns.py

```diff
@@ -0,0 +1,103 @@
+from sys import exc_info
+
+from dis import (
+    cmp_op,
+    dis,
+)
+from enum import (
+    IntEnum,
+    unique,
+)
+
+
+from codetransformer.core import CodeTransformer
+from codetransformer.instructions import (
+    CALL_FUNCTION,
+    CALL_FUNCTION_VAR,
+    POP_TOP,
+    ROT_TWO,
+)
+
+
+@unique
+class Comparisons(IntEnum):
```

Comment:
I think it probably makes sense to move this eventually, but since it's only used here right now, I don't know of a better place yet.  We can always move it later in another patch.